bundles/org.openhab.binding.ipcamera/README.md

```diff
@@ -139,7 +139,7 @@ The discovery feature of openHAB can be used to find and setup ONVIF cameras.
 This method should be preferred as it will discover the cameras IP, ports and URLs for you, making the setup much easier.
 The binding needs to use UDP port 3702 to discover the cameras with, so this port needs to be unblocked by your firewall or add the camera manually if the camera is not auto found.
 To use the discovery, just press the `+` icon located in the Inbox, then select the IpCamera binding from the list of installed bindings.
-The binding will only search using openHAB's currently selected primary network address, see <https://www.openhab.org/docs/settings/>.
+The binding will only search using openHAB's currently selected primary network address, see Main UI>Settings>Network Settings.
```

Comment:
Please revert. We like to guide/link users to the specific documentation. Bindings should nog duplicate that is it will become out of date at some point in time.

Reply:
Broken link has been removed to address the concern.